src/main/java/com/plaid/client/http/ApacheHttpClientHttpDelegate.java

```diff
@@ -52,20 +57,49 @@ public ApacheHttpClientHttpDelegate(String baseUri, CloseableHttpClient httpClie
         	LIBRARY_VERSION = "development version";
         }
     }
-    
+
     public static ApacheHttpClientHttpDelegate createDefault(String baseUri) {
-        CloseableHttpClient httpClient = HttpClients.createDefault();
+
+        return createDefault(baseUri, false);
+    }
+
+    public static ApacheHttpClientHttpDelegate createDefault(String baseUri, boolean disableContentCompression) {
+
+        CloseableHttpClient httpClient = createTLSEnabledHttpClient(disableContentCompression);
         return new ApacheHttpClientHttpDelegate(baseUri, httpClient);
     }
-    
+
+    private static CloseableHttpClient createTLSEnabledHttpClient(boolean disableContentCompression) {
+
+        HttpClientBuilder httpClientBuilder = createTLSEnabledHttpClientBuilder();
+        if (disableContentCompression) {
+            httpClientBuilder.disableContentCompression();
+        }
+        return httpClientBuilder.build();
+    }
+
+    private static HttpClientBuilder createTLSEnabledHttpClientBuilder() {
+
+        SSLContext sslContext = SSLContexts.createDefault();
+        SSLConnectionSocketFactory sf = new SSLConnectionSocketFactory(
+                sslContext,
+                new String[] { "TLSv1.1", "TLSv1.2" },
+                null,
+                SSLConnectionSocketFactory.getDefaultHostnameVerifier());
+
+        return HttpClients
+                .custom()
+                .setSSLSocketFactory(sf);
+    }
```

Comment:
FYI @superwomyn and I paired on this chunk here. In order to land on this somewhat interesting-looking configuration, we just scraped the innards out of this method: `HttpClients.createDefault()` and created out own `SSLConnectionSocketFactory` (also by scraping the innards out of the underlying defaults method.